independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json

```diff
@@ -0,0 +1,597 @@
+    "version": "1.0",
+	"contact": {
+      "name": "Paul Culmsee",
+      "url": "https://www.sevensigma.com.au",
+      "email": "paul.culmsee@sevensigma.com.au"
+    }
+  },
+  "host": "images-api.nasa.gov",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/search": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "collection": {
+                  "type": "object",
+                  "properties": {
+                    "href": {
+                      "type": "string",
+                      "description": "URL of Result Set",
+                      "title": "href"
+                    },
+                    "items": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "data": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "center": {
+                                  "type": "string",
+                                  "description": "NASA center which published the media"
+                                },
+                                "date_created": {
+                                  "type": "string",
+                                  "description": "Media creation date"
+                                },
+                                "description": {
+                                  "type": "string",
+                                  "description": "Media description"
+                                },
+                                "album": {
+                                  "type": "array",
+                                  "items": {
+                                    "type": "string"
+                                  },
+                                  "description": "Albums this media belongs to"
+                                },
+                                "keywords": {
+                                  "type": "array",
+                                  "items": {
+                                    "type": "string"
+                                  },
+                                  "description": "Media keywords"
+                                },
+                                "media_type": {
+                                  "type": "string",
+                                  "description": "Media Type"
+                                },
+                                "nasa_id": {
+                                  "type": "string",
+                                  "description": "The media asset\u2019s NASA ID"

```

Comment:
@paulculmsee Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.